.github/build.yaml.gomplate

```diff
@@ -249,11 +248,11 @@
       - name: Build ISO from local build 🔧
         if: github.event_name != 'schedule' && !startsWith(github.ref, 'refs/tags/')
         run: |
-          sudo -E MAKEISO_ARGS="--name cOS-Seed-{{{ $flavor }}}-${{ env.COS_VERSION }}-{{{ $config.arch }}}" make local-iso
+          sudo -E MAKEISO_ARGS="--name cOS-Seed-{{{ $flavor }}}-${{ env.COS_VERSION }}-{{{ $config.arch }}}{{{- if eq $config.arch "arm64"}}} --repo ${{ env.FINAL_REPO }}{{{end}}}" make local-iso
```

Comment:
This will collide with the fix on #1289 It might result on ignoring the local builds and use only the FINAL_REPO.

IMHO we need to make elemental aware of the build arch, otherwise having default repos will always be tricky as it will use x86 repos for non x86 archs. We need something like what is discussed in https://github.com/rancher-sandbox/cos-toolkit-docs/pull/82#discussion_r860632614 in think that for arm builds the default repository should be the arm repository. We could add the arch field in `v1.Repository` and filter by arch in Luet interface implementation and make the default include all archs. But then we need a way to tell elemental which is the architecture we are building for.

Reply:
An alternative without touching elemental code could be to set the arm repository with the manifest. Using the manifest is the only way to overwrite default repository and with full priority control, so we could add the arm repository in manifest and this would overwrite the default and set the arm repo with a low priority (default prio would be good, currently set to `90`) to give preference on the local repository that `make local-iso` target adds.

Reply:
Yes, this is only a workaround for the current build stuff on arm. I need to be able to build the proper isos for now to see if the other( wrong repo in the cloud config yamls) arm issues are now fixed. This is not meant to be merged as is :)